src/instrumentation/page.ts

```diff
@@ -0,0 +1,85 @@
+import { ReadableSpan } from '@opentelemetry/sdk-trace-base'
+import { Initialiser, setConfig } from '../config'
+import { exportSpans, proxyExecutionContext } from './common'
+import { Exception, SpanKind, SpanOptions, SpanStatusCode, context as api_context, trace } from '@opentelemetry/api'
+import { wrap } from '../wrap'
+import {
+	gatherIncomingCfAttributes,
+	gatherRequestAttributes,
+	gatherResponseAttributes,
+	getParentContextFromRequest,
+} from './fetch'
+
+type PageHandlerArgs = Parameters<PagesFunction>
+
+let cold_start = true
+export function executePageHandler(pagesFn: PagesFunction, [request]: PageHandlerArgs): Promise<Response> {
+	const spanContext = getParentContextFromRequest(request.request)
+
+	const tracer = trace.getTracer('pagesHandler')
+	const attributes = {
+		['faas.trigger']: 'http',
+		['faas.coldstart']: cold_start,
+		['faas.invocation_id']: request.request.headers.get('cf-ray') ?? undefined,
```

Comment:
Could you use semantic attributes import to remain consistent with the rest of the library? Once we update to the latest api, we will be able to tree-shake it to just these three keys being imported.